### PR TITLE
adding sd to the feedstock

### PR DESCRIPTION
**Pull request by ivergara**
Checklist
- [x ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x ] Source is from official source.
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ x] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ivergara** [comment]
@conda-forge-admin, please ping team

File: recipes/sd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% set version = "v0.7.6" %}
```
Comment (xhochy) [inline recipes/sd/meta.yaml:1]: Can you remove the `v` prefix here and add it to the URL?

**Review by xhochy — APPROVED** [review]

**Review by isuruf — CHANGES_REQUESTED** [review]
This doesn't bundle the licenses of rust deps.

**Comment by xhochy** [comment]
@isuruf Do we have anything better than https://github.com/conda-forge/rage-feedstock/blob/8d12868f40b85d3ce9d185ba45a1912b18175afe/recipe/build.sh#L8-L17 nowadays?

**Comment by ivergara** [comment]
@isuruf is there any kind of example on how to accomplish that? Or is what @xhochy pointed out the only way? I used the recipes for `bat` and `ripgrep` as a template.

**Comment by isuruf** [comment]
@xhochy, see https://github.com/conda-forge/conda-forge.github.io/issues/1052

File: recipes/sd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  sha256: faf33a97797b95097c08ebb7c2451ac9835907254d89863b10ab5e0813b5fe5f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```
Comment (isuruf) [inline recipes/sd/meta.yaml:18]: Why do you need this?

Comment (ivergara) [reply recipes/sd/meta.yaml:18]: Not necessarily, but checking the `meta.yml` file in three other packaged rust projects they all have it. I could try removing it and see if the automated tests don't fail.

**Comment by jonashaag** [comment]
Added Rust licenses

File: recipes/sd/meta.yaml (hunk 2 above)
Comment (jonashaag) [reply recipes/sd/meta.yaml]: This is now changed to a copy-pasta from `bat`

**Comment by xhochy** [comment]
LGTM, @isuruf also fine with you?

**Review by isuruf — APPROVED** [review]

**Pull request merged**